### DISCUSSION TITLE
Migrating from SNAP to Docker. Any way to back up and restore without having to re-pair?

### DISCUSSION THREAD

**Discussion by lee-costa** [zwave-js/zwave-js-ui#3271] [General]
I was just wondering if the above is possible. I am looking to move my install from Snap to Docker and would hate to have and re-pair everything. 

Thank you.

**Comment by kpine** [comment]
Even if you don't backup and restore, you don't need to re-pair the devices. That info is stored on the controller.

Easiest way might be to just download a store backup and restore it. Use the buttons in the Store page.

**Reply by lee-costa** [reply]
Thanks for that information. 

I will create a backup via the Store page and import it after installation of Docker.
